src/common/ChannelChatters.hpp

```diff
@@ -17,13 +17,16 @@ class ChannelChatters
     void addRecentChatter(const QString &user);
     void addJoinedUser(const QString &user);
     void addPartedUser(const QString &user);
+    const QColor getUserColor(const QString &user);
     void setChatters(UsernameSet &&set);
+    void setUserColor(const QString &user, const QColor &color);
```

Comment:
Follow-up to comment above - `getUserColor` could be moved right above `setUserColor` to keep things in logical order.